src/commands/project/spec/add.js

```diff
@@ -0,0 +1,55 @@
+const { putProject } = require('../../../requests/project')
+const {
+    getProjectIdentifierArg,
+    getSpecTypeIdentifierArg
+} = require('../../../support/command/parse-input')
+const BaseCommand = require('../../../support/command/base-command')
+
+class AddSpecCommand extends BaseCommand {
+
+    async run() {
+        const { args } = this.parse(AddSpecCommand)
+        const projectPath = getProjectIdentifierArg(args)
+        const specType = getSpecTypeIdentifierArg(args)
+        const specName = args['SPEC_NAME']
+
+        await this.addSpecToProject(projectPath, specType, specName)
+    }
+
+    async addSpecToProject(projectPath, specType, specName) {
+
+        return this.executeHttp({
+            execute: () => putProject({ pathParams: [projectPath, specType, specName] }),
+            onResolve: this.logCommandSuccess({ specName, specType, projectPath }),
+            options: {}
+        })
+    }
+}
+
+AddSpecCommand.description = 'Adds a spec to an existing project.'
+
+
+AddSpecCommand.examples = [
+    'swaggerhub project:spec:add organization/project_name apis my_api',
+    'swaggerhub project:spec:add organization/project_name domains my_domain'
```

Comment:
I don't think 'spec' as part for command is clear. You could go with `project:api:add` / `project:domain:add` rather than combining them into a single command.

Reply:
https://github.com/SmartBear/swaggerhub-cli/pull/228/commits/5f8fddf8aaf37e1f945ea5e8b6ce264f3ac3f2b2